gem/ci/build.ps1

```diff
@@ -17,7 +17,9 @@ function Get-ContainerVersion
         git fetch origin 'refs/tags/*:refs/tags/*'
     }
 
-    (git describe) -replace '-.*', ''
+    $gitver = git describe
+    if ($LASTEXITCODE -ne 0) { return '0.0.0' }
+    $gitver -replace '-.*', ''
```

Comment:
What is this change for?

Reply:
Needed to prevent warnings / errors from showing up when using `Invoke-ContainerTest` in this repo. This repo has no git tags, hence `git describe` fails.

Reply:
Ahhh ok, makes sense why we haven't encountered that yet.